src/components/Hero/Hero.mdx

```diff
@@ -528,4 +527,9 @@ const CampaignFallBackExample = () => {
 
 ## Changelog
 
+| Component Version | DS Version   |
+| ----------------- | ------------ |
+| Added             | `0.2.0`      |
+| Latest            | `Prerelease` |
+
```

Comment:
This can be removed because it is in the revamped page header.

Reply:
I think I saw it somewhere else as the updated v4 pattern so I moved it down here. I'll remove.